src/main/java/conf/Routes.java

```diff
@@ -29,9 +29,12 @@ public void init(Router router) {
         
         router.GET().route("/").with(ApplicationController.class, "index");
 //        router.GET().route("/hello_world.json").with(ApplicationController.class, "helloWorldJson");
-        router.GET().route("/AcesUp").with(ApplicationController.class, "acesUp");
+        router.GET().route("/AcesUp").with(ApplicationController.class, "start");
+        router.GET().route("/AcesUp/engGame").with(ApplicationController.class, "engAcesUp");
```

Comment:
I am not too sure how the controllers work, but is it okay to assume this is how we are linking the front end and models together? Also, thanks George for taking the time to merge all our work together!